### ISSUE TITLE
problem

### ISSUE THREAD

**Issue by suke27** [lizhengwei1992/MS_SSIM_pytorch#1]
for Loss Functions for Neural Networks for Image Processing
it include a*ms_ssim+(1-a)*L1*G
do you realize ms_ssim only?

**Comment by abduallahmohamed** [comment]
@suke27  I garee with you, 
But I think it should be: 
alpha*(1.0-ms_ssim_loss)+(1-alpha)*L1_loss
